SplitOrderHashMap.java

```diff
@@ -192,16 +190,42 @@ public boolean insert(int data) {
    */
   public String toString() {
     String s = "======================================================\nBUCKETS: \n";
-    int i = 0;
-    for (Node bucket : buckets) {
-      String b;
-      if (bucket == null) {
-        b = "null";
-      } else {
-        b = bucket.toString();
+    // int i = 0;
```

Comment:
Please don't leave commented-out stuff. It makes files hard to read. We can always look back or revert using git